scripts/build/build/targets.py

```diff
@@ -483,6 +483,7 @@ def AndroidTargets():
     yield target.Extend('arm64-chip-tool', board=AndroidBoard.ARM64, app=AndroidApp.CHIP_TOOL)
     yield target.Extend('x64-chip-tool', board=AndroidBoard.X64, app=AndroidApp.CHIP_TOOL)
     yield target.Extend('x86-chip-tool', board=AndroidBoard.X86, app=AndroidApp.CHIP_TOOL)
+    yield target.Extend('x86-matter-controller', board=AndroidBoard.X86, app=AndroidApp.MATTER_CONTROLLER)
```

Comment:
If we're eventually going to make this look like chip-tool, I'd rather we name this `java-chip-tool` as opposed to a new 'matter-controller', which doesn't convey that it's only meant for java.

Reply:
we can, but 'chip'  is obsolete term, should we continue to use it?

Reply:
Agree that naming should migrate. chip-tool is currently a `ui-controller-example` and other chip-tool should probably get renamed. CHIP is obsolete and `tool` is very very vague in usage.